Resources/Prototypes/Entities/Objects/Tools/crowbars.yml

```diff
@@ -0,0 +1,128 @@
+- type: entity
+  name: crowbar
+  parent: BaseItem
+  id: BaseCrowbar
+  abstract: true
+  components:
+  - type: EmitSoundOnLand
+    sound:
+      path: /Audio/Items/crowbar_drop.ogg
+  - type: Tag
+    tags:
+    - Crowbar
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    damage:
+      types:
+        Blunt: 8
+    soundHit:
+      collection: MetalThud
+  - type: Tool
+    qualities:
+      - Prying
+    useSound:
+      path: /Audio/Items/crowbar.ogg
+  - type: ToolTileCompatible
+  - type: PhysicalComposition
+    materialComposition:
+      Steel: 100
+  - type: StaticPrice
+    price: 22
+  - type: Prying
+  - type: Clothing
+    quickEquip: false
+    slots:
+    - Belt
+  
+- type: entity
+  parent: BaseCrowbar
+  id: Crowbar
+  description: A multipurpose tool to pry open doors and fight interdimensional invaders.
+  components:
+  - type: Sprite
+    sprite: Objects/Tools/Crowbars/crowbar.rsi
+    state: icon
+  - type: Item
+    size: Normal
+    shape:
+    - 0,0,0,1
```

Comment:
why dont write `state: icon`, size and shape in BaseCrowbar?

Reply:
oh thats true, but i will still have to keep the state: icon because of the rsi change for each crowbar i believe

Reply:
> oh thats true, but i will still have to keep the state: icon because of the rsi change for each crowbar i believe

state icon also is taken from parent